pokemon.go

```diff
@@ -39,6 +40,8 @@ type PokemonData struct {
 	BaseStats  [6]int // base stats of a Pokemon
 	EvYield    [6]int // effort points gained when Pokemon is defeated
 	GrowthRate int
+	IsBiGender bool  // true if this Pokemon must have `Male` or `Female`, false if this Pokemon must be `Genderless`
```

Comment:
Why not `HasGender` or `IsGenderless`?

Reply:
It makes Pokemon genderless by default, which is our previous behavior.